source/components/temp_doeclim_component.cpp

```diff
@@ -422,7 +423,8 @@ void TempDOECLIMComponent::run( const double runToDate ) throw ( h_exception ) {
     
     // Some needed inputs
     //double forcing = core->sendMessage( M_GETDATA, D_RF_TOTAL ).value( U_W_M2 );
-    int tstep = runToDate - core->sendMessage( M_GETDATA, D_START_DATE );
+    //int tstep = runToDate - core->sendMessage( M_GETDATA, D_START_DATE );
```

Comment:
Don't leave commented-out lines of code in the final product.  If we don't need them anymore, delete them.  You don't want to wind up featured on the next episode of Code Hoarders.